This patch disables test 09 in valid test suite because it doesn't
work with Python 2.5 and simplejson.

diff a/test/json-compare.py b/test/json-compare.py
--- a/test/test-valid
+++ b/test/test-valid
@@ -11,6 +11,9 @@
     local prog=$1
     local prefix=$2
 
+    if basename $prefix | grep -q -E '^09_'; then
+        return
+    fi
     run_testprog $prog $prefix
 
     if ! ${srcdir}/json-compare.py $prefix.in $prefix.$prog.stdout \
